runtime/cere-dev/src/lib.rs

```diff
@@ -1482,17 +1501,7 @@ impl Get<Perbill> for NominationPoolsMigrationV4OldPallet {
 }
 
 /// Runtime migrations
-type Migrations = (
-	// 0.9.40
-	pallet_nomination_pools::migration::v4::MigrateV3ToV5<
-		Runtime,
-		NominationPoolsMigrationV4OldPallet,
-	>,
-	// this release they will be properly pruned after the bonding duration has
-	// elapsed)
-	pallet_grandpa::migrations::CleanupSetIdSessionMap<Runtime>,
-	pallet_preimage::migration::v1::Migration<Runtime>,
-);
+type Migrations = (pallet_offences::migration::v1::MigrateToV1<Runtime>,);
```

Comment:
You need to include:
```
runtime_common::session::migration::ClearOldSessionStorage<Runtime>,
``` 
From polkadot, which means that their runtime_common has a custom struct that implements the migration so copy this over.